source/iostreams.tex

```diff
@@ -4335,7 +4405,8 @@
 explicit basic_istream(basic_streambuf<charT, traits>* sb);
 \end{itemdecl}
 
-\indexlibrary{\idxcode{init}!\idxcode{basic_ios}}%
+\indexlibrary{\idxcode{init}!\idxcode{basic_istream}}%
+\indexlibrary{\idxcode{basic_istream}!\idxcode{init}}%
```

Comment:
Reverted to the original, it is indexing a reference to base class initialization in the following text.